### PR TITLE
Add platform info to query string

### PR DESCRIPTION
**Pull request by NandoCT1014**
Adding platform information to query string. Using Unreal Engine platform macros to determine platform. Removed macros for PS4 and PS5 as I couldn't find them in Unreal Engine source code (using  UGameplayStatics::GetPlatformName() to capture playstation platform info).

Also changed the way we get engine information using <%- ueTargetVersion %> instead of  FEngineVersion::Current().ToString(). Seems cleaner and much easier to parse in kusto.

### CODE REVIEW

File: template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs [hunk 1]
```diff
@@ -32,12 +33,12 @@ namespace PlayFabCommon
...
             {
                 return rSettings->ProductionEnvironmentURL
-                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engine=") + engineVersion;
+                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engineVer=") + engineVersion + TEXT("&platform=") + platformString;
             }
             else
             {
                 return TEXT("https://") + rSettings->TitleId + rSettings->ProductionEnvironmentURL
-                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engine=") + engineVersion;
+                    + urlPath + TEXT("?sdk=") + versionString + TEXT("&engineVer=") + engineVersion + TEXT("&platform=") + platformString;
```
Comment (tculotta) [inline template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs]: Is engineVer the correct key here? The Unity PR changed its string from engineVersion to engine.

Comment (tculotta) [reply template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs]: Also the Unity PR didn't include sdk version. Should it?

**Review by tculotta — CHANGES_REQUESTED** [review]

File: template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs (hunk 1 above)
Comment (NandoCT1014) [reply template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs]: Oh no engineVer isn't the correct key here I should change it to match the unity

Comment (NandoCT1014) [reply template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs]: Also the unity sdk already included sdk version info in the query string prior to my recent pr

Comment (tculotta) [reply template/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs]: Hm. I don't see it but I'll follow up not on this unrelated PR.

**Review by tculotta — APPROVED** [review]

**Pull request merged**